vcpkg/ports/qgis/precompiled.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```

Comment:
Is this patch still needed with the `-DUSE_CCACHE=OFF`?

Reply:
No